### ISSUE TITLE
Wrong issue location with rule S1116 on non-css files (html, vue, etc...)

### ISSUE THREAD

**Issue by nicolas-harraudeau-sonarsource** [SonarSource/sonar-css#241]
S1116 raises an issue on the wrong line when it analyses a non-css file.

In the following case the location is correct:
File: index.css
```css
.category-title {
}
.category-filters {
  display: none;
}
.product-listing {
  justify-content: center;; /* Issue here */
}
```

In the following case the issue location is wrong:
File: index.html
```html
<!doctype html>
<html lang="en">
<head>
  <title>Index</title>
  <style>
    .category-title {
    }
    .category-filters { /* Issue raised here. */
      display: none;
    }
    .product-listing {
      justify-content: center;; /* Issue should have been raised here. */
    }
</style>
</head>
<body>
  <?= "Hello World!" ?>
</body>
</html>
```

Reproduced with both `sonar-css-plugin-1.1.1.1010` and `sonar-css-plugin-1.2.0.1317`

**Comment by andrea-guarino-sonarsource** [comment]
issue reported on stylelint repo https://github.com/stylelint/stylelint/issues/5153

**Comment by yassin-kammoun-sonarsource** [comment]
Stylelint decided not to fix the issue and just closed it. Nothing we can do unfortunately.
